trade-finance-manager-ui/server/nunjucks-configuration/filter-formatBooleanAsString.js

```diff
@@ -0,0 +1,13 @@
+const formatBooleanAsString = (bool) => {
+  if (bool) {
+    return 'Yes';
+  }
+
+  if (bool === false) {
+    return 'No';
+  }
+
+  return '-';
+};
```

Comment:
Could technically do this in a few less lines, but really not bothered either way
```
  if (typeof bool === 'boolean') {
    return bool ? 'Yes' : 'No';
  }
  return '-';
```

Reply:
Great point, my brain was not feeling snazzy apparently when it wrote that, now improved!